data.json

```diff
@@ -741,14 +751,18 @@
             "root": "anydo",
             "symlinks": [
                 "any_do",
+                "chrome-ocgddccilgpeepgglnlpchkpgamkgmld-Default"
```

Comment:
You have forgot a comma here

Reply:
I'm so glad you pointed that out, I was looking at the Travis output scratching my head as to what all the errors were 😅